dmutils/presenters.py

```diff
@@ -28,3 +28,10 @@ def _upload(self, value):
             "url": value or "",
             "filename": value.split("/")[-1] or ""
         }
+
+    def _boolean(self, value):
+        if value is True:
```

Comment:
Would that also return `True` if you sent it `"string which isn't blank"` or `1`? Don't think that's desired behaviour.